catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -428,8 +429,28 @@ public Response getDocument(@Context UriInfo uriInfo, @Context HttpServletReques
     @Path("/sources/{sourceid}/{id}")
     public Response getDocument(@Encoded @PathParam("sourceid") String encodedSourceId,
             @Encoded @PathParam("id") String encodedId,
-            @QueryParam("transform") String transformerParam, @Context UriInfo uriInfo,
-            @Context HttpServletRequest httpRequest) {
+            @QueryParam("transform") String transformerParam,
+            @Context UriInfo uriInfo, @Context HttpServletRequest httpRequest) {
+        return getDocument(encodedSourceId, encodedId, transformerParam, false, uriInfo, httpRequest);
+    }
+
+    /**
+     * REST Get. Retrieves the metadata entry specified by the id from the federated source
+     * specified by sourceid. Transformer argument is optional, but is used to specify what format
+     * the data should be returned.
+     *
+     * @param sourceid
+     * @param id
+     * @param transformerParam
+     * @param uriInfo
+     * @return
+     */
+    @GET
+    @Path("/sources/{sourceid}/*/{id}")
```

Comment:
Yes. it is either that or we name it `../anytag/..` Or whatever else someone can think of to signify the one that basically adds a tags is like * to the query.